test/unit/sql/create-table.test.js

```diff
@@ -27,6 +27,31 @@ describe(Support.getTestDialectTeaser('SQL'), function() {
         });
       });
     });
+    if(Support.getTestDialect() === 'postgres') {
+      describe('offers Postgres support', function() {
```

Comment:
Currently this only applies to postgres, but in the future it could apply to other dialects possibly.